### ISSUE TITLE
dist gone?

### ISSUE THREAD

**Issue by plufz** [glidejs/glide#597]
Sorry if this is an old issue. My app have stopped building because of the missing dist directory. Is it by design the dist directory is removed from the repo or am I doing something wrong?

**Comment by joananeves1091** [comment]
how can i use it if i'm not using npm nor sass. Is there a way to still use just with a link to css?

**Comment by jedrzejchalubek** [comment]
You can directly download specific build file it from the release page: https://github.com/glidejs/glide/releases

**Comment by plufz** [comment]
Ah, alright. I'm in a legacy project using bower. That is probably why...

**Comment by jedrzejchalubek** [comment]
Yeah, I think Bower directly uses repositories as source and, yeah, dist folder on the repo has been removed as it is a good practice to not hold output files directly in git history

**Comment by hudri** [comment]
This is an issue for anybody not using NPM. We deliberately keep NodeJS outside of our build toolchain, and just reference github repos/zips in our `composer.json` file. There are only pre-built JS assets on the releases page, but no CSS assets.

Please bring back the pre-built `/dist` folder. It is a a quite common practice in PHP's composer world to include JS libraries directly from git repos.

**Comment by weronikasiuda** [comment]
Same here. Please bring back the dist folder.

**Comment by hudri** [comment]
A note for people looking for a download including the `dist` folder: We are now using `registry.npmjs.org` instead of github.

E.g. in our composer.json for PHP/Drupal we now use

```
{
    repositories: [
        {
            "type": "package",
            "package": {
                "name": "glidejs/glide",
                "version": "3.5.2",
                "type": "drupal-library",
                "dist": {
                    "url": "https://registry.npmjs.org/@glidejs/glide/-/glide-3.5.2.tgz",
                    "type": "tar"
                }
            }
        }    
    ]
}
```

You can get this url using `npm view [package name] dist.tarball` or, in the browser, on the project page on jsdelivr.net, on top of the page, right side of the title, hover over the download icon.

**Comment by xerc** [comment]
@jedrzejchalubek ; please also see https://github.com/cdnjs/packages/pull/1234

**Comment by xerc** [comment]
https://www.npmjs.com/package/glidejs << REMOVE ?
https://www.npmjs.com/package/@glidejs/glide
